.github/workflows/pull-evaluation-tests.yaml

```diff
@@ -0,0 +1,210 @@
+name: "Evaluation tests"
+run-name: "Evaluation tests"
+
+# Required repository secrets:
+# - AICORE_SECRET
+# - COMPANION_CONFIGMAP
+# - GARDENER_TEST_CLUSTER_URL
+# - GARDENER_TEST_CLUSTER_AUTH_TOKEN
+# - GARDENER_TEST_CLUSTER_CA_DATA
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    paths:
+      - "src/**"
+
+# global env variables.
+env:
+  DOCKER_TIMEOUT: 30
+  K3D_VERSION: "v5.7.2" # Required K3d version.
+  REPOSITORY_FULL_NAME: "${{ github.repository }}" # <owner>/<repository-name>.
+  GITHUB_EVENT_ACTION: "${{ github.event.action }}"
+  IMAGE_NAME: "europe-docker.pkg.dev/kyma-project/dev/kyma-companion:PR-${{ github.event.number }}"
+
+jobs:
+  wait-for-build:
+    name: Wait for image build job
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
```

Comment:
nitpick: it'd be better if we use the same python version if possible.

Reply:
This python version is not for the evaluation tests, but for the python script to wait for build job. The evaluation tests setup uses the same python version defined in the poetry project. https://github.com/kyma-project/kyma-companion/blob/bd64043e2436dff44ed175885fe8a50d1e7aeef0/.github/workflows/pull-evaluation-tests.yaml#L154

Reply:
lets keep it fixed for now.